### PR TITLE
Refactor store

### PR DESCRIPTION
**Pull request by doliG**
Solve the problem of having devtools and slice together by splitting slice into separate store

### CODE REVIEW

**Comment by doliG** [comment]
Hey @ferostar can you please approve and merge this one ? Thanks

File: packages/frontend/components/Borrow/Borrow.tsx [hunk 1]
```diff
@@ -105,7 +116,7 @@ export default function Borrow() {
         Switch network
       </Button>
     )
-  } else if (value > 0 && value > balance) {
+  } else if (collateralAmount > 0 && collateralAmount > balance) {
```
[the remark this answers is not in the document]
Comment (doliG) [reply packages/frontend/components/Borrow/Borrow.tsx:119]: No the two value are the same. At first I wanted to have both a number and a string that match each other to do comparison without casting, but it was a bad idea 🙃

[the remark this answers is not in the document]
Comment (ferostar) [reply packages/frontend/components/Borrow/Borrow.tsx:119]: We'll all be living on Mars and will still have issues casting values 🤷‍♂️.

[the remark this answers is not in the document]
Comment (doliG) [reply packages/frontend/components/Borrow/Borrow.tsx:119]: Yup it seems `+value` or Number(value)` will continue to live forever (like wordpress 😬)

**Review by brozorec — APPROVED** [review]

**Pull request merged**